### PR TITLE
Implement prechat timer pop up

### PR DESCRIPTION
**Pull request by tuseto**

### CODE REVIEW

File: src/lib/chat-ui.js [hunk 1]
```diff
@@ -541,11 +552,24 @@ const ChatUi = {
...
+    let actionCodes = actionService.getActionCodes(
+      this.assistant.initialMessage.content,
+      actionService.ACTION_CODE_REGEX,
+    );
+
+    let popUpHtml = getPopUp(actionCodes[0]);
+
+    if (popUpHtml) {
+      this.elements.chatbotContainer.appendChild(popUpHtml);
+      setTimeout(() => {
```
Comment (MiroslavKisov) [inline src/lib/chat-ui.js:565]: @tuseto 
Can't you just use the delay function and omit the setTimeout?

File: src/lib/helpers.js [hunk 2]
```diff
@@ -126,6 +126,11 @@ export const extractStringWithBrackets = (message) => {
   };
 };
 
+export function removeTextBetweenHashtags(input) {
```
Comment (MiroslavKisov) [inline src/lib/helpers.js:129]: @tuseto 
There is a function called "clearButtonCodes" in the action-service.js that does the same, maybe we can use move it here in the helpers.js and remove "removeTextBetweenHashtags" this one.

Comment (tuseto) [reply src/lib/helpers.js]: I moved the function into action-service, but cannot use the old one, they both are doing different things.

File: src/lib/chat-ui.js (hunk 1 above)
Comment (tuseto) [reply src/lib/chat-ui.js:565]: I can, but I don't want to. I even want to remove the delay function.

File: src/lib/helpers.js (hunk 2 above)
Comment (MiroslavKisov) [reply src/lib/helpers.js]: @tuseto 

I was suppose to be the other way around, use the already existing function in the from the action service called (clearButtonCodes) and move it  into helpers.js and then remove the function you wrote.

Comment (tuseto) [reply src/lib/helpers.js]: We can't clear all codes before the message is loaded (when all codes are cleared). It will result in more changes.

**Review by MiroslavKisov — APPROVED** [review]

**Pull request merged**